components/vm/12-interpolated-defaults.tf

```diff
@@ -3,12 +3,25 @@ locals {
     "stg"  = "nle"
     "prod" = "prod"
   }
+
+  xdr_env = var.env == "stg" ? "nonprod" : var.env
```

Comment:
would be cleaner to use this logic directly on line 14 rather than creating a new local just pass in